### PR TITLE
Update Rebel Clash from kirby's repo

### PR DESCRIPTION
**Pull request by scoot404**
Also edited to make sure build errors wouldn't occur (spacing out some of the evolvesFrom), along with fixing the resistances to be -30 instead of -20 from kirby's.

Also marks the unfinished cards as NOT_IMPLEMENTED.

### CODE REVIEW

**Comment by scoot404** [comment]
Build is failing and I can't really understand why based on the error. If either of you could take a look at it and let me know what it appears to be, that'd be appreciated.

File: src/main/resources/cards/431-rebel-clash.yaml [hunk 1]
```diff
@@ -2451,37 +2507,40 @@ cards:
...
+  subTypes: [BASIC, POKEMON_V]
   hp: 160
   retreatCost: 2
-  abilities: 
+  abilities:
   - type: Ability
-    name: Iron Defender
-    text: As long as this Pokemon is in play, damage done to any of your Pokemon with Falinks in its name by your opponent’s atacks is reduced by 20.
-  moves: 
+    name: ''
```
Comment (luongthomas) [inline src/main/resources/cards/431-rebel-clash.yaml:2536]: Looks like CI doesn't like that the name and text for this ability is blank

File: src/main/resources/cards/431-rebel-clash.yaml [hunk 2]
```diff
@@ -3474,581 +3549,665 @@ cards:
...
```
Comment (luongthomas) [inline src/main/resources/cards/431-rebel-clash.yaml:4071]: Name and text for this ability is blank

File: src/main/resources/cards/431-rebel-clash.yaml [hunk 3]
```diff
@@ -749,39 +760,38 @@ cards:
...
+  subTypes: [BASIC, POKEMON_V]
   hp: 210
   retreatCost: 2
-  abilities: 
+  abilities:
   - type: Ability
-    name: Field Runner
-    text: If there is a Stadium card in play, this Pokemon has no Retreat Cost.
-  moves: 
+    name: ''
```
Comment (luongthomas) [inline src/main/resources/cards/431-rebel-clash.yaml:788]: name for this ability is blank

File: src/main/resources/cards/431-rebel-clash.yaml [hunk 4]
```diff
@@ -3474,581 +3549,665 @@ cards:
...
```
Comment (luongthomas) [inline src/main/resources/cards/431-rebel-clash.yaml:3891]: name for this ability is blank

**Comment by scoot404** [comment]
I believe this is ready to be merged now that build errors have been fixed.

**Pull request merged**